### PR TITLE
[docs] Add status header for awsxray exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for awsxray exporter

### CODE REVIEW

**Comment by Frapschen** [comment]
hi,pls have time to review :)

**Comment by Frapschen** [comment]
\cc CODEOWNERS @willarmiros to review this PR

File: exporter/awsxrayexporter/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # AWS X-Ray Tracing Exporter for OpenTelemetry Collector
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
```
Comment (willarmiros) [inline exporter/awsxrayexporter/README.md:7]: Please include the AWS distro: https://aws-otel.github.io/docs/getting-started/x-ray#configuring-the-aws-x-ray-exporter

**Review by willarmiros — COMMENTED** [review]
Lgtm after the comment!

File: exporter/awsxrayexporter/README.md (hunk 1 above)
Comment (Frapschen) [reply exporter/awsxrayexporter/README.md]: @willarmiros done

**Review by willarmiros — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**